### PR TITLE
Doy noy tests

### PR DESCRIPTION
**Pull request by CeciliaNilsson709**
Does this work ok?

### CODE REVIEW

File: tests/testthat/test-doy_noy.R [hunk 1]
```diff
@@ -1,3 +1,28 @@
...
+pvol <- read_pvolfile(pvolfile)
+
 test_that("doy_noy() returns error on incorrect parameters", {
+  expect_error(doy("not_a_object"))
+  expect_error(noy("not_a_object"))
+
+})
+
+test_that("doy_noy() returns a number", {
+  expect_true(is.numeric(doy(vp)))
```
Comment (peterdesmet) [inline tests/testthat/test-doy_noy.R:14]: it is better to use  `expect_type()` to test data types. So here:

    expect_type(doy(vp), "integer")

File: tests/testthat/test-doy_noy.R [hunk 2]
```diff
@@ -1,3 +1,28 @@
...
+
+  expect_true(is.numeric(noy(vp)))
+  expect_true(is.numeric(noy(vpts)))
+  expect_true(is.numeric(noy(vpi)))
+  expect_true(is.numeric(noy(pvol)))
+})
 
+test_that("doy_noy() returns correct doy and noy", {
+  expect_equal(doy(as.POSIXct("2015-01-01 18:00:00 UTC"), 12.8517, 56.3675), 1)
+  expect_equal(noy(as.POSIXct("2015-01-01 18:00:00 UTC"), 12.8517, 56.3675), 2)
```
Comment (peterdesmet) [inline tests/testthat/test-doy_noy.R:27]: Might be good to have another test + comment here to explain:
```suggestion
  # noy starts with New Years eve night as first night
  expect_equal(noy(as.POSIXct("2015-01-01 00:00:00 UTC"), 12.8517, 56.3675), 1) 
  expect_equal(noy(as.POSIXct("2015-01-01 18:00:00 UTC"), 12.8517, 56.3675), 2)
```

**Review by peterdesmet — CHANGES_REQUESTED** [review]
Looks good, see comments

File: tests/testthat/test-doy_noy.R (hunk 1 above)
Comment (CeciliaNilsson709) [reply tests/testthat/test-doy_noy.R:14]: Thats exactly how I wrote it from the start, but covr could not handle the difference between "integer" and "double" so I gave up and made this work around

File: tests/testthat/test-doy_noy.R (hunk 2 above)
Comment (CeciliaNilsson709) [reply tests/testthat/test-doy_noy.R:27]: Sure

**Review by peterdesmet — APPROVED** [review]

**Comment by adokter** [comment]
In US this tests produces an error:
── Failure (test-doy_noy.R:27:3): doy_noy() returns correct doy and noy ────────
doy(as.POSIXct("2015-01-01 18:00:00 UTC"), 12.8517, 56.3675) not equal to 1.

**Comment by adokter** [comment]
Hi @CeciliaNilsson709, see difference below, which caused the test to fail in the US, but not on your local machine in EU:
```
> as.POSIXct("2015-01-01 18:00:00 UTC")
[1] "2015-01-01 18:00:00 EST"
> as.POSIXct("2015-01-01 18:00:00", tz="UTC")
[1] "2015-01-01 18:00:00 UTC"
```
Fixing in PR #463

**Pull request merged**